src/lib/slide-toggle/slide-toggle.scss

```diff
@@ -169,3 +162,15 @@ $mat-slide-toggle-spacing: 8px !default;
   transition-property: background-color;
   transition-delay: 50ms;
 }
+
+// Ripple positioning for the slide-toggle. Moves the ripple container into the center of the thumb.
+.mat-slide-toggle-ripple {
+  position: absolute;
+  top: $mat-slide-toggle-thumb-size / 2 - $mat-slide-toggle-ripple-radius;
+  left: $mat-slide-toggle-thumb-size / 2 - $mat-slide-toggle-ripple-radius;
+  height: $mat-slide-toggle-ripple-radius * 2;
+  width: $mat-slide-toggle-ripple-radius * 2;
+  border-radius: 50%;
+  z-index: 1;
+  pointer-events: none;
+}
```

Comment:
@dahaupt good question, I never thought about why that's the convention we use, but apparently that's just how lines are defined: http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline